ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/questionnaire-completed/questionnaire-completed.component.html

```diff
@@ -3,12 +3,116 @@
     <h1 class="govuk-heading-xl" i18n="@@questionnaireCompleted_h1">
       Questionnaire completed
     </h1>
+    <div>
+      <app-hearing-details-header></app-hearing-details-header>
+    </div>
+    <div>
+      <p class="govuk-body">Here's a summary of the information you've provided.</p>
+    </div>
+
+    <h2 class="govuk-heading-m">About you</h2>
+    <table class="govuk-table" style="width:100%;">
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYou.answer}">
+          Is there anything that could affect your ability to take part in a video hearing?
+        </td>
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYou.answer}" style="text-align:right;">
+          <strong>{{ model.aboutYou.answer | appBooleanHumanView }}</strong></td>
+      </tr>
+      <tr *ngIf="model.aboutYou.answer">
+        <td class="govuk-table__cell" colspan="2">
+          <span class="govuk-caption-m">{{model.aboutYou.notes}}</span>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    <h2 class="govuk-heading-m">About your location</h2>
+    <table class="govuk-table" style="width:100%;">
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">
```

Comment:
for all these texts, can you also add in a `i18n` attribute? you'll find examples in other places for this

Reply:
done